sdk/keyvault/keyvault-certificates/samples/javascript/importCertificate.js

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const { CertificateClient, WellKnownIssuer } = require("@azure/keyvault-certificates");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+// Load the .env file if it exists
+require("dotenv").config();
+
+// For convenience in this sample we'll use some self-signed test certificates
+// that were generated using openssl
+const samplePfxBase64 =
```

Comment:
Looks suspicious but overall harmless - I just copied the same self signed test cert that dotnet uses instead of generating a different one https://github.com/Azure/azure-sdk-for-net/blob/311f34ea6ee203adb787b0319adf0415fbc30839/sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/samples/Sample3_ImportCertificate.cs#L99-L204


Debated between inlining these and making PEM and PFX files - but those are gitignored so I figured it's not worth constantly having to force add them. Let me know if you think otherwise!